controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -863,7 +863,8 @@ public void authTokenCheckFailed(WireCommands.AuthTokenCheckFailed authTokenChec
      * @param scope               Stream scope.
      * @param stream              Stream name.
      * @param keys                List of {@link TableKey}s to be removed. Only if all the elements in the list has version as
-     *                            {@link KeyVersion#NOT_EXISTS} then an unconditional update/removal is performed. Else an atomic conditional
+     *                            {@link KeyVersion#NO_VERSION} then an unconditional update/removal is performed. Else an atomic
+     *                            conditional
```

Comment:
Please fix the Javadoc indentation on these two lines.

Reply:
done.